modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-disk-partitioning_{context}"]
+= Configuring disk partitioning with SiteConfig
+
+You can use the `SiteConfig` YAML file to generate the `MachineConfig` CR used for disk partitioning. Prior to installation, you need to modify values in the `SiteConfig` CR to reflect dependencies on the underlying disk.
```

Comment:
The CR is the resource that you patch/push to the hub cluster. The YAML file is just a representation of that resource. So the YAML files doesn't do anything on it's own, you apply it as a resource on the cluster to make the cluster updates. 


```suggestion
Use a `SiteConfig` CR to generate the `MachineConfig` CR used for disk partitioning. Prior to installation, you need to modify values in the `SiteConfig` CR to reflect dependencies on the underlying disk.
```

Reply:
Done.